defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+# List of groups managed by ansible. E.g.
+#
+# .. code-block:: YAML
+#
+#   groups_group:
+#     - name: wheel
+#       gid: 1000
+#       system: true
+#       state: present
+#
+# The default group list is empty.
+groups_group: []
```

Comment:
Can you add an example?

Reply:
isn't the example which is commented out enough?

Reply:
Sorry man, I'm still blind. Should get more coffee before doing reviews